articles/iot-develop/concepts-digital-twin.md

```diff
@@ -24,17 +24,11 @@ The example IoT Plug and Play device in this article implements a [Temperature C
 
 ## Device twins and digital twins
 
-As well as a digital twin, Azure IoT Hub also maintains a *device twin* for every connected device. A device twin is similar to a digital twin in that it's a representation of a device's properties. The Azure IoT service SDKs include APIs for interacting with device twins.
```

Comment:
Consider **provisioned** rather than **connects**
`An IoT hub initializes a digital twin and a device twin, when the first time an IoT Plug and Play device is provisioned`

Reply:
I made this change and will merge the pull request.